contrib/checkpoint/checkcriu.go

```diff
@@ -0,0 +1,11 @@
+package main
```

Comment:
Missing the copyright boilerplate

Reply:
Also needs a comment line to explain what this tool does.

Reply:
Added both